### ISSUE TITLE
Just download, but don't install

### ISSUE THREAD

**Issue by gaborcsardi** [MangoTheCat/remotes#41]
Would be nice.

**Comment by krlmlr** [comment]
How's this different from #34?

**Comment by gaborcsardi** [comment]
It would only download packages, without installing them.

**Comment by krlmlr** [comment]
Are you going to support downloading to a cranlike? That would resolve the better part of #34, and an issue I'm experiencing with updating packages used in the current session ([AppVeyor run](https://ci.appveyor.com/project/krlmlr/styler/build/1.0.43#L282)).

**Comment by gaborcsardi** [comment]
FWIW there is `remotes::download_version` that downloads a certain version, by default the latest.

**Comment by krlmlr** [comment]
With dependencies?

**Comment by gaborcsardi** [comment]
No. You can get CRAN-like dependencies with `tools::package_dependencies`,
if that's enough for you.

**Comment by cderv** [comment]
I am getting dependencies with `remotes::package_deps` : should I change to `tools::package_dependencies` ? 
And using `remotes::download_version`, why download to a tempfile ? Would it be possible to make a where arg to download to a specific folder keeping package name ?  

My use case is the same : I am trying to download to a cranlike.
